src/data.js

```diff
@@ -6940,6 +6940,41 @@ module.exports = [
     phone: 'iphone',
     tags: ['Designer', 'Developer', 'Front End', 'JavaScript', 'React'],
   },
+  {
+    name: 'George Campbell',
+    description: 'Full stack engineer at Netflix',
+    url: 'https://gibr.net/uses',
```

Comment:
Ping to "https://gibr.net/uses" failed with status: 404

Reply:
i fixed the redirect.